src/Microsoft.Azure.WebJobs.Host/Bindings/TraceWriter/TraceWriterBindingProvider.cs

```diff
@@ -31,8 +28,7 @@ public Task<IBinding> TryCreateAsync(BindingProviderContext context)
             }
 
             ParameterInfo parameter = context.Parameter;
-            if (parameter.ParameterType != typeof(TraceWriter) &&
-                parameter.ParameterType != typeof(TextWriter))
```

Comment:
>TextWriter [](start = 50, length = 10)

This is a significant breaking change! ... (I use it frequently in my own projects) . What's the recommended pattern that people switch to?

Reply:
I've removed `TextWriter` -- but `TraceWriter` is still usable. Do we need to keep supporting `TextWriter` going forward? This is a chance to remove old stuff if we want to. We can keep it, but we've had some bugs in the past with this and I wasn't sure anyone used it anymore.

Reply:
If you use TraceWriter or ILogger from within the function, it will still log to the dashboard. Or does TextWriter have some kind of behavior that's unique?

For the sake of this PR, what about if I add it back (it should be fairly straightforward) and we can follow up later to see if it's worth pulling out?

Reply:
That'd be fine. Thanks.